resources/views/ad-detail/main.twig

```diff
@@ -239,6 +241,7 @@
                     </div>
                 {% endif %}
             </div>
+                    {{ addBlock('ad-detail/title/bottom-action', {'adv_id':adv.id, 'sahibinden':true})|raw }}
```

Comment:
replace with bottom code for just call single module

`{{ addBlock('ad-detail/title/bottom-action', {'adv_id':adv.id, 'sahibinden':true}, ['complaints'])|raw }}`